Makefile

```diff
@@ -208,28 +209,31 @@ run: generate manifests	## Run against the configured Kubernetes cluster in ~/.k
 	go run ./main.go
 
 install: manifests	## Install CRDs into a cluster
-	kubectl apply -f config/crd/bases
+	kubectl apply -f deployment/network-operator/crds/
 
 uninstall: manifests	## Uninstall CRDs from a cluster
-	kubectl delete -f config/crd/bases
+	kubectl delete -f deployment/network-operator/crds/
 
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${TAG}
 	$(KUSTOMIZE) build config/resources-namespace | kubectl apply -f -
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
+	kubectl apply -f hack/crds/*
```

Comment:
what is hack/crds ?

Reply:
It contains `networkattachmentdefinitions` CRD